bin/shundle

```diff
@@ -316,7 +316,7 @@ _shundle_install()
             _shundle_install__element="${_shundle_install__array%%,*}"
             _shundle_install_aux "${_shundle_install__element}"
             [ X"${_shundle_install__array}" = X"${_shundle_install__element}" ] &&\
-                ${_shundle_install__array}='' || _shundle_install__array="${_shundle_install__array#*,}"
+                "${_shundle_install__array}"='' || _shundle_install__array="${_shundle_install__array#*,}"
```

Comment:
I think this is invalid syntax, never though that VAR="foobar" was equal to "VAR"="foobar", I think the quotes should go around the value, not the variable